pyopensn/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+# Created on Sat, February 22
+#
+# Copyright (c) 2025 quocdang1998
+
+# Find Python3
+find_package(Python3 COMPONENTS Interpreter Development REQUIRED)
+
+# Get pybind11 CMake directory and append it to CMake prefix path
+execute_process(
+  COMMAND ${Python3_EXECUTABLE} -m pybind11 --cmakedir
+  OUTPUT_VARIABLE PYBIND11_CMAKE_DIR
+  OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+list(APPEND CMAKE_PREFIX_PATH "${PYBIND11_CMAKE_DIR}")
+
+# Execute Python to get the mpi4py include directory.
+execute_process(
+    COMMAND ${Python3_EXECUTABLE} -c "import mpi4py; print(mpi4py.get_include())"
+    OUTPUT_VARIABLE MPI4PY_INCLUDE_DIR
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+
+# Locate Pybind11
+find_package(pybind11 CONFIG)
+
+# Construct module
+pybind11_add_module(__init__
+    src/aquad.cpp
+    src/env.cpp
+    src/fieldfunc.cpp
+    src/logvol.cpp
+    src/main.cpp
+    src/mat.cpp
+    src/mesh.cpp
+    src/parameter.cpp
+    src/solver.cpp
+    src/source.cpp
+    src/xs.cpp
+)
```

Comment:
You may want to switch this to `file(GLOB_RECURSE <VAR> CONFIGURE_DEPENDS...`. It is what the main CMakeLists.txt uses. People usually forget to add a new file into CMake and then they wonder what is going on. That's why we prefer the `GLOB`